requirements.txt

```diff
@@ -3,3 +3,4 @@ datasets
 torchaudio
 soundfile
 librosa
+fairseq2
```

Comment:
As it's early stage of fairseq2 shall we add version to avoid compatibility issues?

Reply:
Sg, fixed